docs/windows_installer_tutorial/AutomatedInstallerWorkflowSetup.md

```diff
@@ -0,0 +1,150 @@
+# Setting up a Windows Installer SBOM Generation Workflow
+
+## Summary
+When working with Windows software installers, it's often not obvious where the installer places its files. With this setup, the installer is ran in a modified VirtualBox VM that has a file system driver installed. This tool makes a guess at what files came from the installer and extracts them from the VM into the host machine. The extracted files are then placed into their full paths on the host, except instead of the C: drive, they're placed into a folder named "C" which is entirely empty besides those files extracted from the VM. Running Surfactant on this folder essentially creates SBOMs for almost any installer.<br><br>Following these instructions result in a working VM. The mentioned process has been made entirely automatic by running a single command.

```

Comment:
```suggestion
When working with Windows software installers, it's often not obvious where the installer places its files. With this setup, the installer is run in a modified VirtualBox VM that has a file system driver installed. This tool uses the file system driver to determine what files came from the installer and extracts them from the VM into the host machine. The extracted files are then placed into a directory structure on the host that mimics the guest file system VM, except instead of the `C:` drive, the files are placed into a subfolder named `C` which only contains those files extracted from the VM. Running Surfactant on this folder can be essentially be used to create SBOMs for almost any installer.

Following these instructions result in a working VM. The mentioned process has been made entirely automatic by running a single command.

```